### PR TITLE
Add dnsdumpster in dns_search sources

### PR DESCRIPTION
**Pull request by adviksinghania**
For issue #82 
Added dnsdumpster to sources in modules/osint/dns_search.py

### CODE REVIEW

File: modules/osint/dns_search.py [hunk 1]
```diff
@@ -14,6 +14,8 @@
 
 import re
 import json
+import requests
```
Comment (saeeddhqan) [inline modules/osint/dns_search.py]: don't use requests.
Instead use self.request('url').text

File: modules/osint/dns_search.py [hunk 2]
```diff
@@ -165,6 +167,40 @@ def rapiddns(self, q):
...
+		'sec-fetch-site': 'same-origin',
+		'sec-fetch-mode': 'navigate',
+		'sec-fetch-user': '?1',
+		'sec-fetch-dest': 'document',
+		'referer': 'https://dnsdumpster.com/',
+		'accept-language': 'en-US,en;q=0.9',
+		'cookie': cookie
+	}
+	try:
+		soup = BeautifulSoup(init_res.content, 'html.parser')
```
Comment (saeeddhqan) [inline modules/osint/dns_search.py]: Don't use bs4.
re.search(r'<input type="hidden" name="csrfmiddlewaretoken" value="(.*)">', req.text)...

File: modules/osint/dns_search.py [hunk 3]
```diff
@@ -165,6 +167,40 @@ def rapiddns(self, q):
 		j = self.page_parse(req.text).get_dns(q) or []
 		set_data(j)
 
+def dnsdumpster(self, q):
+	self.verbose('[DNSDUMPSTER] Searching in dnsdumpster...')
+	init_res = requests.get('https://dnsdumpster.com/')  # initial response from dnsdumpster
+	set_cookie = init_res.headers.get('Set-Cookie')  # getting crsftoken from cookie field
+	cookie = set_cookie[:set_cookie.index(';') + 1]
+	headers = {
```
Comment (saeeddhqan) [inline modules/osint/dns_search.py:173]: I think you don't need to add all of the fields to the headers.
Remove the unnecessary fields.

**Review by saeeddhqan — COMMENTED** [review]
Check it out.
@z3y50n will help you.

**Comment by adviksinghania** [comment]
I have made the necessary changes. Kindly review again.
Thanks.

**Comment by saeeddhqan** [comment]
Great.

**Pull request merged**